api/v1/checks.go

```diff
@@ -382,6 +382,32 @@ type Mongo struct {
 	MongoDBCheck `yaml:",inline" json:",inline"`
 }
 
+/*
+[include:datasources/elasticsearch_pass.yaml]
+*/
+
+type Elasticsearch struct {
+	ElasticsearchCheck `yaml:",inline" json:",inline"`
+}
+
+type ElasticsearchCheck struct {
+	Description `yaml:",inline" json:",inline"`
+	URL         string `yaml:"url" json:"url,omitempty" template:"true"`
+
```

Comment:
Can we also add: `Templatable `yaml:",inline" json:",inline"`` in the struct. This gives more flexibilty in terms of tests with expr and JS... also can display it nicely on the dashboard

Reply:
Sure